GenderPayGap.WebUI/Scripts/Application/LiveSearch.js

```diff
@@ -118,10 +118,12 @@
 
     LiveSearch.prototype.showLoadingIndicator = function showLoadingIndicator() {
         this.$loadingBlock.text('Loading...');

```

Comment:
When is this displayed?

Reply:
Not sure whether it is ever shown with the 'Loading...' variant. It may be if the search functionality is being slow. The '&nbsp;' variant from line 125 is definitely show, and causes misalignment without the added this.$loadingBlock.hide(); on line 126. Currently it is here:
![image](https://user-images.githubusercontent.com/59561751/113739144-69bcc700-96f7-11eb-86d6-6e0452080fe6.png)